2.13/tests/test/configs/config_mock.json

```diff
@@ -3,106 +3,54 @@
+    "authentication": "basic",
+    "cases": [
         {
-            "instance_id":"",
-            "body": {
-                "plan_id": "d3031751-XXXX-XXXX-XXXX-a42377d3320e", 
-                "service_id" :"acb56d7c-XXXX-XXXX-XXXX-feb140a59a66",                                
-                "parameters": {
-                    "billing-account": "fake-account"
+            "name": "Mock Service Test",
+            "service_id" :"acb56d7c-XXXX-XXXX-XXXX-feb140a59a66",
+            "organization_guid": "org-guid-here",
+            "space_guid": "space-guid-here",
+            "lifecycle": [
+                {
+                    "operation": "provision",
+                    "plan_id": "d3031751-XXXX-XXXX-XXXX-a42377d3320e",
+                    "async": true,
+                    "parameters": {
+                        "billing-account": "fake-account"
+                    }
                 },
-                "organization_guid": "org-guid-here",
-                "space_guid": "space-guid-here" 
-            },            
-            "scenario": "new",
-            "notes": "Fake service instance",
-            "async": true
-        },
-        {
-            "instance_id":"EXISTING_ID",
-            "body": {
-                "plan_id": "d3031751-XXXX-XXXX-XXXX-a42377d3320e", 
-                "service_id" :"acb56d7c-XXXX-XXXX-XXXX-feb140a59a66",                                
-                "parameters": {
-                    "billing-account": "fake-account"
-                },
-                "organization_guid": "org-guid-here",
-                "space_guid": "space-guid-here" 
-            },            
-            "scenario": "new",
-            "notes": "Fake service instance",
-            "async": true
-        },
-        {
-            "instance_id":"EXISTING_ID",
-            "body": {
-                "plan_id": "0f4008b5-XXXX-XXXX-XXXX-dace631cd648", 
-                "service_id" :"acb56d7c-XXXX-XXXX-XXXX-feb140a59a66",                                
-                "parameters": {
-                    "billing-account": "fake-account"
+                {
+                    "operation": "update",
+                    "plan_id": "0f4008b5-XXXX-XXXX-XXXX-dace631cd648",
+                    "async": true,
+                    "parameters": {
+                        "parameter1":1,
+                        "parameter2":"foo"
+                    }
                 },
-                "organization_guid": "org-guid-here",
-                "space_guid": "space-guid-here" 
-            },            
-            "scenario": "conflict",
-            "notes": "Fake service instance",
-            "async": true
-        }
-    ],
-    "updates":[
-        {
-            "instance_id":"",
-            "body":{
-                "context" :{
-                    "platform": "cloudfoundry",
-                    "some_field": "some-contextual-data"
+                {
+                    "operation": "bind",

```

Comment:
Don't it need some optional fields such like `context` and `bind_resource`?

Reply:
The field is optional and the test framework doesn't add any case for it. Any time we can add it back for new cases which require it. Does it make sense?